sdks/go/pkg/beam/core/graph/coder/coder.go

```diff
@@ -177,6 +177,7 @@ const (
 	Iterable           Kind = "I"
 	KV                 Kind = "KV"
 	LP                 Kind = "LP" // Explicitly length prefixed, likely at the runner's direction.
+	IWCValue           Kind = "IWCvalue"
```

Comment:
Note how nothing else in this `coder` package calls out it's coderness in their short names. I'd just call this `IntervalWindow` with IW as the short version.

The 'value' sufixes for WindowedValue and ParamWindowedValue indicate they're wrapping a value themselves, rather than being windows themselves, like the one we're adding here.

Reply:
I had named it `IntervalWindow` to start, but there's a conflicting `IntervalWindow` value define [here](https://github.com/camphillips22/beam/blob/c83b42f7547a5fd57e2f9a90d7b613c512eb088c/sdks/go/pkg/beam/core/graph/coder/windows.go#L25-L25). Maybe just `IW` for const name with a comment that explicitly calls out that it refers to an `IntervalWindow`? Any other suggestions?

I can definitely change the value to "IW" though.